tests/test-amp-img-sanitizer.php

```diff
@@ -12,42 +12,62 @@ public function setUp() {
 
 	public function get_data() {
 		return array(
-			'no_images' => array(
+			'no_images'                                => array(
 				'<p>Lorem Ipsum Demet Delorit.</p>',
 				'<p>Lorem Ipsum Demet Delorit.</p>',
 			),
 
-			'image_without_src' => array(
+			'image_without_src'                        => array(
 				'<p><img width="300" height="300" /></p>',
 				'<p></p>',
 			),
 
-			'image_with_empty_src' => array(
+			'image_with_empty_src'                     => array(
 				'<p><img src="" width="300" height="300" /></p>',
 				'<p></p>',
 			),
 
-			'image_with_self_closing_tag' => array(
+			'image_with_empty_width'                   => array(
+				'<p><img src="http://placehold.it/300x300" width="" height="300" /></p>',
+				'<p><amp-img src="http://placehold.it/300x300" width="600" height="300" class="amp-wp-unknown-size amp-wp-unknown-width amp-wp-enforced-sizes" sizes="(min-width: 600px) 600px, 100vw"></amp-img></p>',
+			),
+
+			'image_with_empty_height'                  => array(
+				'<p><img src="http://placehold.it/300x300" width="300" height="" /></p>',
+				'<p><amp-img src="http://placehold.it/300x300" width="300" height="400" class="amp-wp-unknown-size amp-wp-unknown-height amp-wp-enforced-sizes" sizes="(min-width: 300px) 300px, 100vw"></amp-img></p>',
+			),
+
+			'image_with_zero_width'                    => array(
+				'<p><img src="http://placehold.it/300x300" width="0" height="300" /></p>',
+				'<p><amp-img src="http://placehold.it/300x300" width="0" height="300" sizes="(min-width: 0px) 0px, 100vw" class="amp-wp-enforced-sizes"></amp-img></p>',
+			),
```

Comment:
Please also include a test with a zero width _and_ height so we can ensure that the [support topic](https://wordpress.org/support/topic/amp-img-not-setting-height-and-width/) is covered:

```html
<img src="https://example.com/img.jpg" width="0" height="0" border="0">
```

Reply:
Could you add a test for this?

Reply:
Done ✔️